[OSDOCS-5672]: Deprecated k8s topo labels (#58098)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

[id="ocp-4-13-mco-certificate-changes"]
==== The MCO now syncs kubelet CA certificates on paused pools

> Previously, the Machine Config Operator (MCO) updated the kubelet client certificate authority (CA) certificate, `/etc/kubernetes/kubelet-ca.crt`, as a part of the regular machine config update. Starting with {product-title} (product-version}, the `kubelet-ca.crt` no longer gets updated as a part of the regular machine config update. As a result of this change, the Machine Config Daemon (MCD) automatically keeps the `kubelet-ca.crt` up to date whenever changes to the certificate occur.

Reviewer: header says MCO syncs the CA, but this paragraph seems to be saying it no longer does so because MCD does instead?

Reviewer: (unrelated to this change i guess, but confusing regardless)

Author: Don't know about that one, tagging @mburke5678 who added this line

Reviewer: i think the only  change needed is to switch the header from "The MCO syncs......" to "The MCD syncs..."

but yes, it would be nice to fix it.

Author: Gonna resolve this so it's collapsed since it's tangential to the change here and sounds like y'all have an agreement :slightly_smiling_face:

Reviewer 2: @bparees PTAL: https://github.com/openshift/openshift-docs/pull/58103

* `platform.vsphere.ingressVIP`
* `platform.vsphere.network`

[id="ocp-4-13-node-dep-topology-labels"]
==== Kubernetes topology labels

> The commonly used Kubernetes topology labels `failure-domain.beta.kubernetes.io/zone` and `failure-domain.beta.kubernetes.io/region` have been replaced with `topology.kubernetes.io/zone` and `topology.kubernetes.io/region`, respectively. The replacement labels are available starting with Kubernetes 1.17 and {product-title} version 4.4.

Reviewer 3: ```suggestion
Two commonly used Kubernetes topology labels are being replaced. The `failure-domain.beta.kubernetes.io/zone` label is replaced with `topology.kubernetes.io/zone`. The `failure-domain.beta.kubernetes.io/region` label is replaced with `topology.kubernetes.io/region`. The replacement labels are available starting with Kubernetes 1.17 and {product-title} version 4.4.
```
IBM Style lists "respectively" as a do-not-use term, so I tried to rewrite this part for you to avoid that word. (See the [Word Usage page](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r), and Ctrl+F to "respectively".)

> Currently, both the deprecated and replacement labels are supported, but support for the deprecated labels is planned to be removed in a future release. To prepare for the removal, you can update your clusters to use the newer labels for any topology-based affinities that you have configured.

Reviewer 4: What's the term that we generally use for the stuff customers add to their clusters? Do we call that workload? Cluster integrations? That's the stuff they need to update. The cluster itself sounds like the thing that we're on the hook to maintain and should be taking care of for them.

I'm fine with this as is, but if we have a commonly used term for that I think we should use that term rather than "update your clusters"

Reviewer: how about something like: "modify any resources(e.g. volumes, deployments, other workloads) that reference the old labels to instead use the new labels"

?

Author: > What's the term that we generally use for the stuff customers add to their clusters?

Great question, and not something I end up writing about enough to know. I will solicit input from the larger docs team on that one :slightly_smiling_face:

Author: Seems like "workloads" is the consensus term here :+1:

Author: Hey Ben - sorry, I did not see you comment (probably because I did not refresh the page). Will revise with that input!

Author: revised - lmk it you think it works better now

Reviewer 4: looks good to me.

Reviewer: lgtm
